doc/getting-started.md

```diff
@@ -0,0 +1,344 @@
+{
+  "hex": "01000000013324faf8f03695261611669f5bdd93b68a86cf02db12dc46642ed50666ed69b8000000006a47304402200e1bf722d4335179de170f7c762755b463b3f7b8f026f30950f701bc834f0e6e022036295fdd5e607ca41c4e0e62e59d0911b607bfabedde2424665ffae13564d0e001210388f8f226d12eccd3ba93c1454ec4498b065cea96e29b918fbdb517872ebbf581ffffffff0200a5459b010000001976a91418a89ee36293f15c4db4c01173babd579243161188ac60b8c4b8000000001976a914c6977da37560e1432c2e14e16952981a4c272cac88ac00000000",
+  "complete": true
+}
+```
+
+##### sendrawtransaction
+
+Finally, to broadcast the transaction to the network and to let the miners confirm it shortly :
+
+```bash
+# Syntax
+$ dogecoin-cli sendrawtransaction signed_transaction
+
+# Example
+$ dogecoin-cli sendrawtransaction 01000000013324faf8f03695261611669f5bdd93b68a86cf02db12dc46642ed50666ed69b8000000006a47304402200e1bf722d4335179de170f7c762755b463b3f7b8f026f30950f701bc834f0e6e022036295fdd5e607ca41c4e0e62e59d0911b607bfabedde2424665ffae13564d0e001210388f8f226d12eccd3ba93c1454ec4498b065cea96e29b918fbdb517872ebbf581ffffffff0200a5459b010000001976a91418a89ee36293f15c4db4c01173babd579243161188ac60b8c4b8000000001976a914c6977da37560e1432c2e14e16952981a4c272cac88ac00000000
+b4fae2a43cb35f8016a547e9658e061f1da4a043efafecc42f739d46d95dee21
+```
+
+So much spending power !
+
+### Inspect blocks and transactions
+
+Blocks and transactions are identified by unique *hashes*.
+To understand how to retrieve informations, let's try to find the *[coinbase transaction](https://www.javatpoint.com/coinbase-transaction)* of the 69th (actually 70th) block.
+
+To get details of the 69th block :
+
+```bash
+# Find block hash from his height
+$ dogecoin-cli getblockhash 69
+3d2def20cd0d3aca148741ef469bda11647a3040d7669c82745d03c728706a8b
+
+# Get block data
+$ dogecoin-cli getblock 3d2def20cd0d3aca148741ef469bda11647a3040d7669c82745d03c728706a8b
+{
+  "hash": "3d2def20cd0d3aca148741ef469bda11647a3040d7669c82745d03c728706a8b",
+  "confirmations": 7816,
+  "strippedsize": 190,
+  "size": 190,
+  "weight": 760,
+  "height": 69,
+  "version": 1,
+  "versionHex": "00000001",
+  "merkleroot": "695ce4208fa7a87ef9e99805b0910dc129058ecdceb5cef7e25f71dcdc7936db",
+  "tx": [
+    "695ce4208fa7a87ef9e99805b0910dc129058ecdceb5cef7e25f71dcdc7936db"
+  ],
+  "time": 1386475225,
+  "mediantime": 1386475209,
+  "nonce": 3923708672,
+  "bits": "1e0ffff0",
+  "difficulty": 0.000244140625,
+  "chainwork": "0000000000000000000000000000000000000000000000000000000004600460",
+  "previousblockhash": "ffa69e04f928b84f19d84da25fb544340e54dca6c03c33930da245719e61c5ea",
+  "nextblockhash": "44bf8abbbb96d4dcfb95df563e606c37987133ea3e013b23bbddde8d7f905fdd"
+}
+```
+
+See the `tx` field to find the list of all block transactions. Only one exist in the block 69th, a coinbase transaction.
+
+To get transaction details :
+
+```bash
+$ dogecoin-cli getrawtransaction 695ce4208fa7a87ef9e99805b0910dc129058ecdceb5cef7e25f71dcdc7936db
```

Comment:
You can add a boolean to getrawtransaction to get the json:

```
Arguments:
1. "txid"      (string, required) The transaction id
2. verbose       (bool, optional, default=false) If false, return a string, otherwise return a json object
```

PS: are you running with `-txindex`?

Reply:
I wasn't running with `-txindex`. You would like to see the json display ?

Reply:
You're doing 2 commands for something that can be done in one... what does the hex encoded transaction add to someone getting started?

Reply:
Are you speaking about `gettransaction` ? Could you perform your desired command on testnet for `695ce4208fa7a87ef9e99805b0910dc129058ecdceb5cef7e25f71dcdc7936db` and share the output. My pi node crashed with `-txindex`, I'm not sure if I can run it today.

Reply:
`gettransaction` is for wallet tx, `getrawtransaction` is for mempool tx, or if using `-txindex`, all mined+mempool tx.

Assuming you meant mainnet and not testnet, because there is no such tx on testnet.

Without `-txindex`:
```
$ dogecoin-cli getrawtransaction 695ce4208fa7a87ef9e99805b0910dc129058ecdceb5cef7e25f71dcdc7936db
error code: -5
error message:
No such mempool transaction. Use -txindex to enable blockchain transaction queries. Use gettransaction for wallet transactions.
```

With `-txindex`:
```
$ dogecoin-cli getrawtransaction 695ce4208fa7a87ef9e99805b0910dc129058ecdceb5cef7e25f71dcdc7936db
01000000010000000000000000000000000000000000000000000000000000000000000000ffffffff0e04d9eea3520101062f503253482fffffffff0100ac6156be23000023210340a42a5ad6c4c0cd5ae539657032e0a359bd3e0f95771f34d71691b13460a624ac00000000
```

With `-txindex` and `verbose=1` (last arg):
```
$ dogecoin-cli getrawtransaction 695ce4208fa7a87ef9e99805b0910dc129058ecdceb5cef7e25f71dcdc7936db 1
{
  "hex": "01000000010000000000000000000000000000000000000000000000000000000000000000ffffffff0e04d9eea3520101062f503253482fffffffff0100ac6156be23000023210340a42a5ad6c4c0cd5ae539657032e0a359bd3e0f95771f34d71691b13460a624ac00000000",
  "txid": "695ce4208fa7a87ef9e99805b0910dc129058ecdceb5cef7e25f71dcdc7936db",
  "hash": "695ce4208fa7a87ef9e99805b0910dc129058ecdceb5cef7e25f71dcdc7936db",
  "size": 109,
  "vsize": 109,
  "version": 1,
  "locktime": 0,
  "vin": [
    {
      "coinbase": "04d9eea3520101062f503253482f",
      "sequence": 4294967295
    }
  ],
  "vout": [
    {
      "value": 393004.00000000,
      "n": 0,
      "scriptPubKey": {
        "asm": "0340a42a5ad6c4c0cd5ae539657032e0a359bd3e0f95771f34d71691b13460a624 OP_CHECKSIG",
        "hex": "210340a42a5ad6c4c0cd5ae539657032e0a359bd3e0f95771f34d71691b13460a624ac",
        "reqSigs": 1,
        "type": "pubkey",
        "addresses": [
          "D5TjotnkoXekNQBCn54CBWxjEzCJMMe7WS"
        ]
      }
    }
  ],
  "blockhash": "3d2def20cd0d3aca148741ef469bda11647a3040d7669c82745d03c728706a8b",
  "confirmations": 3964556,
  "time": 1386475225,
  "blocktime": 1386475225
}
```

Reply:
Thanks, updated.

Reply:
Yeah probably, I think I used it and I missed adding this information. If you query you own transaction, would you need `txindex` ?

We may change the introduction sentence to explain it, and suggest `txindex` if user want to query other txs.

Reply:
I added a warning about txindex at the top of the section, explaining to use this option if the query is not related to user wallet.